internal/acceptance/steps.go

```diff
@@ -13,6 +14,18 @@ import (
 	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
 )
 
+// ApplyStep returns a Test Step which applies a Configuration and then check that the
+// resource exists. This doesn't do any other assertions since it's expected that an
+// ImportStep will be called afterwards to validate that.
+func (td TestData) ApplyStep(config func(data TestData) string, testResource types.TestResource) resource.TestStep {
```

Comment:
As [here](https://github.com/hashicorp/terraform-provider-azurerm/pull/13801#issuecomment-954458718) not going to do that for now since there's larger changes needed to make that more usable